index.html

```diff
@@ -21,6 +21,10 @@
             <div id='selectionbox'>
                 <div id='room-details'>
                     <input type='text' id='room-name' placeholder='Room Name (Optional)' />
+                    <select id='shard'>
```

Comment:
I've read in slack that there is an endpoint that returns the current shard information (I don't know the exact url though). It might be a good idea to call that, so we won't have to update the page every time a new shard is released.